src/App.jsx

```diff
@@ -14,33 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABET = 'alphabet';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function prepeareGoods(goods, sortField, isReversed) {
+  const prepearedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    prepearedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_FIELD_LENGTH:
+          return good1.length - good2.length;
+        case SORT_FIELD_ALPHABET:
+          return good1.localeCompare(good2);
+        default:
+          return 0;
+      }
+    });
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (isReversed) {
+    prepearedGoods.reverse();
+  }
+
+  return prepearedGoods;
+}
```

Comment:
According to the checklist, method names should start with a verb. Consider renaming 'prepeareGoods' to 'prepareGoods' or 'getPreparedGoods'. This will make the code more readable and understandable.